### ISSUE TITLE
"Unknown function: getcurpos" occurs when using JCimportXXX in older Vim 

### ISSUE THREAD

**Issue by msfm** [artur-shaik/vim-javacomplete2#190]
The following ERROR "Unknown function: getcurpos" occurs when using JCimportXXX (JCimportAdd, JCimportAddI, JCimportsAddMissing and JCimportsRemoveUnused) in Vim 7.4.160, which is included in CentOS/RHEL 7: 

~~~
Error detected while processing function javacomplete#imports#Add..<SNR>67_SortImports:
line   11:
E117: Unknown function: getcurpos
E15: Invalid expression: getcurpos()
line   21:
E121: Undefined variable: saveCursor
E116: Invalid arguments for function setpos
~~~

~~~
Error detected while processing function javacomplete#imports#RemoveUnused:
line    6:
E117: Unknown function: getcurpos
E15: Invalid expression: getcurpos()
line   19:
E121: Undefined variable: saveCursor
E15: Invalid expression: saveCursor[1] - len(unused)
line   20:
E121: Undefined variable: saveCursor
E116: Invalid arguments for function setpos
~~~

`getcurpos()` is not available in 7.4.160 because it wasn't added until 7.4.313. I think using `getpos('.')` instead of `getcurpos()` resolve this issue.

**Comment by wsdjeg** [comment]
@msfm 
I have no this func,but I can use JCimportAdd,why?
![2016-01-31 10 15 12](https://cloud.githubusercontent.com/assets/13142418/12699788/91b23dbc-c803-11e5-93f3-767229505ebb.png)
![2016-01-31 10 15 24](https://cloud.githubusercontent.com/assets/13142418/12699790/92f7bf1c-c803-11e5-8a2a-52e37b211c68.png)

**Comment by msfm** [comment]
Although the error occurs, adding import statement itself would work. However, your cursor position is moved to the line at the added import statement. Because the function (`getcursor()`) is called to preserve the original cursor position and get back your cursor position after adding import statement.

**Comment by wsdjeg** [comment]
my cursor will go back to origin position!

**Comment by msfm** [comment]
I'm not sure why your cursor goes back to original position without having `getcurpos()` func and I don't know what's happening on your environment. 

Anyway, the error message "Unknown function: getcurpos" was resolved in my environment by the PR#191 which I sent.

**Comment by wsdjeg** [comment]
I also feal strange,but now I remove the old vim ,and compile from the source,now I am using 7.4.1222

**Comment by artur-shaik** [comment]
Fixed by #191
